src/test/java/se/sundsvall/smloader/integration/openemapper/precedenceofreemployment/PrecedenceOfReemploymentProviderTest.java

```diff
@@ -42,6 +46,12 @@ class PrecedenceOfReemploymentProviderTest {
 	@Mock
 	private PartyClient partyClientMock;
 
+	@Mock
+	private LabelsProvider labelsProvider;
+
+	@Mock
+	private CaseMetaDataRepository caseMetaDataRepository;
```

Comment:
caseMetadataRepository-> caseMetadataRepositoryMock?

In order to keep the same naming convention as the rest of the class memebers?